lib/renderer/init.ts

```diff
@@ -65,9 +65,9 @@ require('@electron/internal/renderer/common-init');
 
 if (nodeIntegration) {
   // Export node bindings to global.
-  const { makeRequireFunction } = __non_webpack_require__('internal/modules/helpers');
+  const { makeRequireFunction } = __non_webpack_require__('internal/modules/helpers') as typeof import('@node/lib/internal/modules/helpers');
   global.module = new Module('electron/js2c/renderer_init');
-  global.require = makeRequireFunction(global.module);
+  (global.require as any) = makeRequireFunction(global.module);
```

Comment:
Shouldn't this as any be on the rhs, interesting this worked I've never seen it on lhs before